1.pre_process_data/notebooks/0.fix_pathing.ipynb

```diff
@@ -0,0 +1,97 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook fixes issues with data paths by replacing spaces with underscores"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pathlib\n",
+    "\n",
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# set data path\n",
+    "data_path = pathlib.Path(\"../../data\").resolve(strict=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[PosixPath('/home/lippincm/Documents/4TB/data/live_cell_timelapse_apoptosis/data/20231017ChromaLive_1st timepoint of 6hr_4ch'),\n",
+       " PosixPath('/home/lippincm/Documents/4TB/data/live_cell_timelapse_apoptosis/data/20231017ChromaLive_1st timepoint of 6hr_4ch_MaxIP'),\n",
+       " PosixPath('/home/lippincm/Documents/4TB/data/live_cell_timelapse_apoptosis/data/20231004ChromaLive6hr_4ch_MaxIP'),\n",
+       " PosixPath('/home/lippincm/Documents/4TB/data/live_cell_timelapse_apoptosis/data/20231017ChromaLive_6hr_4ch_partial_upload'),\n",
+       " PosixPath('/home/lippincm/Documents/4TB/data/live_cell_timelapse_apoptosis/data/20231017ChromaLive_endpoint w AnnexinV_2ch'),\n",
+       " PosixPath('/home/lippincm/Documents/4TB/data/live_cell_timelapse_apoptosis/data/20231017ChromaLive_6hr_4ch_MaxIP'),\n",
+       " PosixPath('/home/lippincm/Documents/4TB/data/live_cell_timelapse_apoptosis/data/20231017ChromaLive_6hr_4ch'),\n",
+       " PosixPath('/home/lippincm/Documents/4TB/data/live_cell_timelapse_apoptosis/data/20230920ChromaLiveTL_24hr4ch_MaxIP'),\n",
+       " PosixPath('/home/lippincm/Documents/4TB/data/live_cell_timelapse_apoptosis/data/20231017ChromaLive_endpoint w AnnexinV_2ch_MaxIP'),\n",
+       " PosixPath('/home/lippincm/Documents/4TB/data/live_cell_timelapse_apoptosis/data/20230920ChromaLiveTL_24hr4ch'),\n",
+       " PosixPath('/home/lippincm/Documents/4TB/data/live_cell_timelapse_apoptosis/data/20231004ChromaLive6hr_4ch')]"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# get all directories in data path\n",
+    "directories = [x for x in data_path.iterdir() if x.is_dir()]\n",
+    "directories"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# rename the directories to replace spaces with underscores\n",
+    "for directory in directories:\n",
+    "    new_name = directory.name.replace(\" \", \"_\")\n",
+    "    directory.rename(directory.parent / new_name)"
+   ]
+  }
```

Comment:
Would it make sense to run cell 3's content again after this one to help demonstrate the changes took place as desired?

Reply:
added